src/events/app/index.ts

```diff
@@ -0,0 +1,144 @@
+import { Context, Probot, ProbotOctokit } from "probot";
+import { IPullService } from "../../services/PullService";
+import { sleep } from "../../utils/util";
+
+interface RepoConfig {
+  owner: string;
+  repo: string;
+}
+
+// Triggered when the program starts.
+export async function handleAppStartUpEvent(
+  app: Probot,
+  gc: InstanceType<typeof ProbotOctokit>,
+  pullService: IPullService
+) {
+  let repoConfigs: RepoConfig[];
+
+  if (process.env.SYNC_REPOS !== undefined) {
+    repoConfigs = await getSyncRepositoryListFromEnv();
+  } else {
+    repoConfigs = await getSyncRepositoryListFromInstallation(app);
+  }
+
+  for (let repoConfig of repoConfigs) {
```

Comment:
Please use foreach.

Reply:
I don't quite understand why?

Reply:
I think it is better to use foreach to traverse.

Reply:
foreach seems not suitable for use with await. I also need to adjust whether to use await through testing.

<img width="618" alt="Snipaste_2021-01-13_15-40-38" src="https://user-images.githubusercontent.com/5086433/104421158-edae5980-55b5-11eb-8dd7-0d12377f5e66.png">

Reply:
Got it.